src/aics-image-viewer/components/CellViewerCanvasWrapper/index.tsx

```diff
@@ -69,28 +72,34 @@ const ViewerWrapper: React.FC<ViewerWrapperProps> = (props) => {
     return noImageText || spinner;
   };
 
-  const { appHeight, changeViewerSetting, visibleControls, numSlices, numTimesteps, viewMode, region, slice, time } =
+  const { appHeight, changeViewerSetting, visibleControls, numTimesteps, numScenes, viewMode, region, slice, time } =
     props;
+  const clippingPanelTall = numTimesteps > 1 && numScenes > 1 && viewMode === ViewMode.threeD;
```

Comment:
This check happens in two places... For DRY, should it just be a single flag that's passed in as a prop?

Reply:
My thought here was that, since this property is derivable from other passed properties, it was better not to pass it in explicitly and risk an "invalid" value. Though this value has less of a strong notion of "invalid" than, say, a `string` along with an `isAllCaps: boolean` value. So maybe it would be better?